pkg/config/config.go

```diff
@@ -132,24 +137,28 @@ type Configuration interface {
 
 // configuration stores the configuration
 type configuration struct {
-	mux                         sync.RWMutex
-	filters                     []filter
-	excludeGroupRole            []string
-	excludeUsername             []string
-	restrictDevelopmentUsername []string
-	webhooks                    []WebhookConfig
-	generateSuccessEvents       bool
-	reconcilePolicyReport       chan<- bool
-	updateWebhookConfigurations chan<- bool
+	defaultRegistry               string
+	enableDefaultRegistryMutation bool
+	excludeGroupRole              []string
+	excludeUsername               []string
+	filters                       []filter
+	generateSuccessEvents         bool
+	mux                           sync.RWMutex
+	reconcilePolicyReport         chan<- bool
+	restrictDevelopmentUsername   []string
+	updateWebhookConfigurations   chan<- bool
+	webhooks                      []WebhookConfig
 }
 
 // NewConfiguration ...
 func NewConfiguration(client kubernetes.Interface, reconcilePolicyReport, updateWebhookConfigurations chan<- bool) (Configuration, error) {
 	cd := &configuration{
-		reconcilePolicyReport:       reconcilePolicyReport,
-		updateWebhookConfigurations: updateWebhookConfigurations,
-		restrictDevelopmentUsername: []string{"minikube-user", "kubernetes-admin"},
-		excludeGroupRole:            defaultExcludeGroupRole,
+		defaultRegistry:               "docker.io",
+		enableDefaultRegistryMutation: true,
```

Comment:
This is the default value. It will be set to false only if the `enableDefaultRegistryMutation: "false"` is supplied in the config map. 

https://github.com/kyverno/kyverno/pull/4715/files#diff-a3d824da3c42420cd5cbb0a4a2c0e7b5bfddd819652788a0596d195dc6e31fa5R303